docs/configuration/settings.rst

```diff
@@ -5,113 +5,469 @@ Mautic Configuration Settings
+
+* **Append tracking pixel into Email body?** - To track Email opens, select **Yes**. Select **No** to prevent tracking, reporting on, and using decisions based on Email opens.
+
+* **Convert embed images to Base64** - Select **Yes** to display embedded images in Emails as code rather than as images.
+
+* **Disable trackable URLs** - Removes tracking from URLs in your Emails. Select Yes to prevent tracking, reporting on, and using decisions based on link clicks. Some Email service providers don’t like redirecting URLs. Using trackable URLs in your Emails may impact deliverability.
+  
 Unsubscribe settings
 ====================
 
+.. image:: images/unsubscribe-settings.png
+  :width: 600
+  :alt: Screenshot showing Unsubscribe Settings Configuration in Mautic
+
+* **Text for the {unsubscribe_text} token** -  Like the ``{webview_text}`` token, customize the **Unsubscribe** link. For example:
+
+Edit between the ``<a href="|URL|">`` and ``</a>`` tags. Don’t change the URL as it's tokenized. If you add ``{unsubscribe_url}`` as a token in the Email, you won’t see this text.
+
+* **Unsubscribed and resubscribed confirmation message** - If a Contact unsubscribes or resubscribes, this message displays on the page after the respective action. Don’t edit the ``|EMAIL|`` or the ``|URL|`` token in the ``<a href>`` tag.
+
+* **Show Contact preference settings** - Select **Yes** to direct the unsubscribe link to your configured preference center. If you haven’t created a preference center, Mautic creates a default page based on the next 5 settings. The created page uses the default Theme for styling.
+
+* **Show Contact Segment preferences** - Select **Yes** to enable a Contact to change which Segments they’re part of on the preference center page. Segments won’t display on the preference center page if they aren’t published and public.
+
+* **Show Contact frequency preferences** - Select **Yes** to enable an individual to limit the number of marketing messages they receive on each Channel from the preference center.
+
+* **Show pause Contact preferences** - Select **Yes** to enable Contacts to turn-off messages from your Mautic account to their Email address for a specified date range. This action isn’t a full unsubscribe, and at the end of the date range, the message is sent to that address once again.
+
+* **Show Contact’s Categories** - If you have Categories set for Contacts, Campaigns, Emails, etc., select Yes to enable the Contact to opt out of the Categories they choose from the preference center page.
+
+* **Show Contact’s preferred Channel option** - If you have multiple Channels available within your Mautic account (Email, SMS, mobile push, web notifications, etc.), Contacts can choose their preferred Channel. This can be useful if you are using the Marketing Messages feature of Mautic. More information about the Preference Center is available :doc:`here</contacts/preference_center>`.
```

Comment:
```suggestion
* **Show Contact’s preferred Channel option** - If you have multiple Channels available within your Mautic instance (Email, SMS, mobile push, web notifications, etc.), Contacts can choose their preferred Channel. This can be useful if you are using the Marketing Messages feature of Mautic. More information about the Preference Center is available :doc:`here</contacts/preference_center>`.
```